java/src/main/java/svix/SvixOptions.java

```diff
@@ -0,0 +1,19 @@
+package svix;
+
+public class SvixOptions {
+	private final boolean debug;
+	private final String testUrl;
+
+	public SvixOptions(boolean debug, String testUrl) {
```

Comment:
Ideally we'd be able to set only one of these options and have the others still be the default value.
Not sure the best way to do this in java but in Go we'd use pointers and only change the value if explicitly set
i.e. `new SvixOptions(true, null)` => `debug: true, testUrl: https://api.svix.com`

Reply:
Done, added overload for SvixOptions and handled null (just in case)

Reply:
I think the right way of doing it is by doing a builder-like structure. So init things to a certain value by default, and make it possible to do things like:
```
options = new SvixOptions().setDebug(true).setTestUrl("http...");
```
See what I mean?

Reply:
Done. I didn't create another class for the Builder though. (followed your suggestion above)

Reply:
Yeah, no need for a real builder. That's why I said builder-like. :P